test/streaming.js

```diff
@@ -34,3 +36,107 @@ test('Streaming to Twitch', async t => {
   await app.client.waitForExist('button=End Stream', 10 * 1000);
   t.pass();
 });
+
+
+test('Streaming to Twitch', async t => {
+
+  // login into the account
+  if (!(await logIn(t, 'twitch'))) return;
+  const app = t.context.app;
+
+  // open EditStreamInfo window
+  await focusMain(t);
+  await app.client.click('button=Go Live');
+
+  // set stream info, and start stream
+  await focusChild(t);
+  await fillForm(t, 'editStreamForm', {
+    stream_title: 'SLOBS Test Stream',
+    game: 'PLAYERUNKNOWN\'S BATTLEGROUNDS'
+  });
+  await app.client.click('button=Confirm & Go Live');
+
+  // check we're streaming
+  await focusMain(t);
+  await app.client.waitForExist('button=End Stream', 10 * 1000);
+  t.pass();
+});
+
+
+test('Streaming to Facebook', async t => {
+
+  // login into the account
+  if (!(await logIn(t, 'facebook'))) return;
+  const app = t.context.app;
+
+  // open EditStreamInfo window
+  await focusMain(t);
+  await app.client.click('button=Go Live');
+
+  // set stream info, and start stream
+  await focusChild(t);
+  await fillForm(t, 'editStreamForm', {
+    stream_title: 'SLOBS Test Stream',
+    game: 'PLAYERUNKNOWN\'S BATTLEGROUNDS',
+    stream_description: 'SLOBS Test Stream Description'
+  });
+
+  await app.client.click('button=Confirm & Go Live');
+
+  // check we're streaming
+  await focusMain(t);
+  await app.client.waitForExist('button=End Stream', 10 * 1000);
+  t.pass();
+});
+
+// TODO: the auth-token for Mixer get stale very fast
+// test('Streaming to Mixer', async t => {
```

Comment:
Can we `test.skip` this as opposed to just commenting out?

Reply:
Honestly when I see a message about skipped tests in AppVeyour that makes me nervous